src/js/apps/globals/nav/nav_items.js

```diff
@@ -0,0 +1,150 @@
+import Backbone from 'backbone';
+import Radio from 'backbone.radio';
+
+import { i18n } from 'js/views/globals/app-nav/app-nav_views';
+
+const appNavMenu = new Backbone.Collection([
+  {
+    onSelect() {
+      window.open('https://help.roundingwell.com/');
+    },
+    text: i18n.mainNav.help,
+    icon: {
+      type: 'far',
+      icon: 'life-ring',
+    },
+  },
+  {
+    onSelect() {
+      Radio.request('auth', 'logout');
+    },
+    text: i18n.mainNav.signOut,
+    icon: {
+      type: 'fas',
+      icon: 'right-from-bracket',
+    },
+  },
+]);
+
+const adminNavMenu = new Backbone.Collection([
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'dashboards:all');
+    },
+    id: 'DashboardsApp',
+    text: i18n.adminNav.dashboards,
+    icon: {
+      type: 'far',
+      icon: 'gauge',
+    },
+  },
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'programs:all');
+    },
+    id: 'ProgramsApp',
+    text: i18n.adminNav.programs,
+    icon: {
+      type: 'far',
+      icon: 'screwdriver-wrench',
+    },
+  },
+  {
+    onSelect() {
+      Radio.trigger('event-router', 'clinicians:all');
+    },
+    id: 'CliniciansApp',
+    text: i18n.adminNav.clinicians,
+    icon: {
+      type: 'far',
+      icon: 'users-gear',
+    },
+  },
+]);
+
+const patientsAppWorkflowsNav = new Backbone.Collection([
+  {
+    id: 'owned-by',
+    text: i18n.patientsAppNav.ownedBy,
+    icons: [{
+      type: 'fas',
+      icon: 'list',
+    }],
+    event: 'worklist',
+    eventArgs: ['owned-by'],
+  },
+  {
+    id: 'schedule',
+    text: i18n.patientsAppNav.schedule,
+    icons: [{
+      type: 'fas',
+      icon: 'calendar-star',
+    }],
+    event: 'schedule',
+    eventArgs: [],
+  },
+  {
+    id: 'shared-by',
+    text: i18n.patientsAppNav.sharedBy,
+    icons: [{
+      type: 'fas',
+      icon: 'arrow-right-arrow-left',
+    }],
+    event: 'worklist',
+    eventArgs: ['shared-by'],
+  },
+  {
+    id: 'new-past-day',
+    text: i18n.patientsAppNav.newPastDay,
+    icons: [
+      {
+        type: 'fas',
+        icon: 'angle-left',
+      },
+      {
+        type: 'fas',
+        icon: '1',
+      },
+    ],
+    event: 'worklist',
+    eventArgs: ['new-past-day'],
+  },
+  {
+    id: 'updated-past-three-days',
+    text: i18n.patientsAppNav.updatedPastThree,
+    icons: [
+      {
+        type: 'fas',
+        icon: 'angle-left',
+      },
+      {
+        type: 'fas',
+        icon: '3',
+      },
+    ],
+    event: 'worklist',
+    eventArgs: ['updated-past-three-days'],
+  },
+  {
+    id: 'done-last-thirty-days',
+    text: i18n.patientsAppNav.doneLastThirty,
+    icons: [
+      {
+        type: 'fas',
+        icon: '3',
+      },
+      {
+        type: 'fas',
+        icon: '0',
+      },
+    ],
+    event: 'worklist',
+    eventArgs: ['done-last-thirty-days'],
+  },
+]);
+
+export {
+  appNavMenu,
+  adminNavMenu,
+  patientsAppWorkflowsNav,
+};
```

Comment:
Moved these to their own file and exported them. This way I can import them into both the `nav_app.js` and `worklist_app.js` files.

Better place to put these?? Wasn't 100% sure where to put them.